hs_restclient/__init__.py

```diff
@@ -449,18 +480,72 @@ def getScienceMetadata(self, pid):
             <rdfs1:label>Generic</rdfs1:label>
             <rdfs1:isDefinedBy>http://www.hydroshare.org/terms</rdfs1:isDefinedBy>
           </rdf:Description>
         </rdf:RDF>
+
+        Example of data JSON returned:
+
+        {
+            "title":"Great Salt Lake Level and Volume",
+            "creators":[
+                        {"name":"John Smith","description":"/user/24/","organization":"USU","email":"john.smith@usu.edu","address":"Engineering Building, USU, Logan, Utah","phone":"435-789-9087","homepage":null,"order":1},
+                        {"name":"Lisa Miller","description":null,"organization":null,"email":null,"address":null,"phone":null,"homepage":null,"order":2}
+                       ],
+            "contributors":[
+                            {"name":"Jenny Parker","description":"","organization":"Univesity of Utah","email":"jenny_parker@hotmail.com","address":"","phone":"","homepage":""}
+                           ],
+            "coverages":[
+                            {"type":"period","value":{"start":"01/01/2000","end":"12/12/2010"}}
+                        ],
+            "dates":[
+                        {"type":"created","start_date":"2017-01-03T17:06:18.932217Z","end_date":null},
+                        {"type":"modified","start_date":"2017-01-03T17:06:19.162694Z","end_date":null}
+                    ],
+            "description":"Time series of level, area and volume in the Great Salt Lake. Volume and area of the Great Salt Lake are derived from recorded levels",
+            "formats":[
+                        {"value":"image/tiff"}
+                      ],
+            "funding_agencies":[
+                                {"agency_name":"National Science Foundation","award_title":"Model Execution Cyberinfrastructure ","award_number":"NSF_9087658_2017","agency_url":"http://www.nsf.gov"}
+                               ],
+            "identifiers":[
+                            {"name":"hydroShareIdentifier","url":"http://www.hydroshare.org/resource/87ffb608900e407ab4b67d30c93b329e"}
+                        ],
+            "language":"eng",
+            "rights":"This resource is shared under the Creative Commons Attribution CC BY. http://creativecommons.org/licenses/by/4.0/",
+            "type":"http://www.hydroshare.org/terms/GenericResource",
+            "publisher":null,"sources":[],
+            "subjects":[
+                        {"value":"NSF"},
+                        {"value":"Modeling"}
+                       ],
+            "relations":[]
+        }
         """
-        url = "{url_base}/scimeta/{pid}/".format(url_base=self.url_base,
-                                                 pid=pid)
+        if data_format.lower() == 'xml':
+            url = "{url_base}/scimeta/{pid}/".format(url_base=self.url_base, pid=pid)
+        else:
+            url = "{url_base}/resource/{pid}/scimeta/elements".format(url_base=self.url_base, pid=pid)
```

Comment:
@alvacouch Looking at the new rest endpoint implementation for science metadata it seems it would return data only in JSON format. @aphelionz Am I right on this?